packages/bogue-tutorials/bogue-tutorials.0.2.0/opam

```diff
@@ -0,0 +1,44 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Bogue Tutorials"
+description: "Several tutorials for learning how to create GUIs with Bogue"
+maintainer: ["San Vũ Ngọc"]
+authors: ["San Vũ Ngọc"]
+license: ["CC-BY-NC-4.0" "Apache-2.0"]
+tags: ["gui" "bogue" "tutorial"]
+homepage: "https://github.com/sanette/bogue-tutorials"
+doc: "https://sanette.github.io/bogue-tutorials/bogue-tutorials/index.html"
+bug-reports: "https://github.com/sanette/bogue-tutorials/issues"
+depends: [
+  "dune" {>= "3.2"}
+  "bogue" {>= "20240224"}
+  "odoc" {with-doc}
+]
```

Comment:
```suggestion
  "conf-perl"
]
```
For
```

#=== ERROR while compiling bogue-tutorials.0.2.0 ==============================#
# context              2.2.0~beta2~dev | linux/x86_64 | ocaml-base-compiler.5.1.1 | pinned(https://github.com/sanette/bogue-tutorials/archive/0.2.0.tar.gz)
# path                 ~/.opam/5.1/.opam-switch/build/bogue-tutorials.0.2.0
# command              ~/.opam/opam-init/hooks/sandbox.sh build make mld
# exit-code            2
# env-file             ~/.opam/log/bogue-tutorials-7-f34186.env
# output-file          ~/.opam/log/bogue-tutorials-7-f34186.out
### output ###
# make -C common mld
# make[1]: Entering directory '/home/opam/.opam/5.1/.opam-switch/build/bogue-tutorials.0.2.0/common'
# make -C src mld
# make[2]: Entering directory '/home/opam/.opam/5.1/.opam-switch/build/bogue-tutorials.0.2.0/common/src'
# ../make_mld.sh common.ml
# Converting common.ml to common.mld
# ../make_mld.sh: line 51: perl: command not found
# make[2]: *** [Makefile:14: common.mld] Error 127
# make[2]: Leaving directory '/home/opam/.opam/5.1/.opam-switch/build/bogue-tutorials.0.2.0/common/src'
# make[1]: *** [Makefile:9: mld] Error 2
# make[1]: Leaving directory '/home/opam/.opam/5.1/.opam-switch/build/bogue-tutorials.0.2.0/common'
# make: *** [Makefile:18: common] Error 2
```

Reply:
Didn't help enough...
```

#=== ERROR while compiling bogue-tutorials.0.2.0 ==============================#
# context              2.2.0~beta2~dev | linux/x86_64 | ocaml-base-compiler.5.1.1 | pinned(https://github.com/sanette/bogue-tutorials/archive/0.2.0.tar.gz)
# path                 ~/.opam/5.1/.opam-switch/build/bogue-tutorials.0.2.0
# command              ~/.opam/opam-init/hooks/sandbox.sh build make mld
# exit-code            2
# env-file             ~/.opam/log/bogue-tutorials-7-051733.env
# output-file          ~/.opam/log/bogue-tutorials-7-051733.out
### output ###
# make -C common mld
# make[1]: Entering directory '/home/opam/.opam/5.1/.opam-switch/build/bogue-tutorials.0.2.0/common'
# make -C src mld
# make[2]: Entering directory '/home/opam/.opam/5.1/.opam-switch/build/bogue-tutorials.0.2.0/common/src'
# ../make_mld.sh common.ml
# Converting common.ml to common.mld
# sed: unrecognized option: z
# BusyBox v1.36.1 (2023-11-07 18:53:09 UTC) multi-call binary.
# 
# Usage: sed [-i[SFX]] [-nrE] [-f FILE]... [-e CMD]... [FILE]...
# or: sed [-i[SFX]] [-nrE] CMD [FILE]...
# 
# 	-e CMD	Add CMD to sed commands to be executed
# 	-f FILE	Add FILE contents to sed commands to be executed
# 	-i[SFX]	Edit files in-place (otherwise write to stdout)
# 		Optionally back files up, appending SFX
# 	-n	Suppress automatic printing of pattern space
# 	-r,-E	Use extended regex syntax
# 
# If no -e or -f, the first non-option argument is the sed command string.
# Remaining arguments are input files (stdin if none).
# make[2]: *** [Makefile:14: common.mld] Error 1
# make[2]: Leaving directory '/home/opam/.opam/5.1/.opam-switch/build/bogue-tutorials.0.2.0/common/src'
# make[1]: *** [Makefile:9: mld] Error 2
# make[1]: Leaving directory '/home/opam/.opam/5.1/.opam-switch/build/bogue-tutorials.0.2.0/common'
# make: *** [Makefile:18: common] Error 2
```